LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift

```diff
@@ -139,6 +139,7 @@ private struct BasalRateGuardrailWarning: View {
         assert(!crossedThresholds.isEmpty)
 
         let caption = self.isZeroUnitRateSelectable && crossedThresholds.allSatisfy({ $0 == .minimum })
+            // ANNA TODO: ask MLee about this one
```

Comment:
Hey @novalegraReal quick, just wanted to make sure mlee was ok with the wording on this guardrail before we proceed to test? I saw Janet's comment in the card, but wasn't sure if it referred to this or not. TIA :)

Reply:
@ginnyyadav : Anna's (@novalegra ) last day was Friday.  While I think she's still got her email, I think she's backpacking right now.  I recommend following up with @mlee directly.